src/main/java/Point.java

```diff
@@ -22,6 +22,10 @@ public static double angle(Point a, Point b, Point c){
     // I create two vectors from the vertex
     // https://stackoverflow.com/questions/21483999/using-atan2-to-find-angle-between-two-vectors
 
+    if (a.equals(c) ||b.equals(a) ||b.equals(c)){
+      return -1;
+    }
+
```

Comment:
It's almost correct, but I think it should be 
```java
if (b.equals(a) ||b.equals(c)){
```
since it's OK if a == c (none of these are vertices)